utils/find-overlay-dependencies-loop.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+SCRIPT="$(dirname "$0")/find-overlay-dependencies.sh"
+
+# Don't update XCTest
+for overlay in $(find ./stdlib/public/SDK/ -depth 1 -type d ! -name XCTest -exec basename \{\} \;); do
```

Comment:
Should this be $0-relative? (Alternately, should it complain if you're not in a directory where `./stdlib/public/SDK/` exists?)